src/components/ui/Base/index.ts

```diff
@@ -13,5 +13,6 @@ export interface BaseProps extends SpaceProps {
   className?: string;
 }
 
-export const baseStyles = ({ css }: BaseProps) => (css ? `${css};` : '');
+export const baseStyles = ({ css }: BaseProps) =>
+  css ? `@media { ${css} };` : '';
```

Comment:
Is there a way to write responsive media queries in the css prop?

Reply:
Yeah you should be able to still pass in media queries as part of the `css` string